packaging/nntrainer.spec

```diff
@@ -45,7 +45,7 @@
 
 Name:		nntrainer
 Summary:	Software framework for training neural networks
-Version:	0.1.1
+Version:	1.0.0
```

Comment:
i wonder how about release 1.0 once devel headers are a bit finalized? and release 0.2.0 for now?

Reply:
then.. we will do 1.0.0 which includes.
  . finalize devel headers
  . recurrent net
  . tflite model support
  . and more.